keyvault/client/secrets/README.md

```diff
@@ -0,0 +1,337 @@
+
+This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information see the Code of Conduct FAQ or contact opencode@microsoft.com with any additional questions or comments.
+
+<!-- LINKS -->
+[source_code]: https://github.com/Azure/azure-sdk-for-java/tree/master/keyvault/client/secrets/src
+[package]:https://mvnrepository.com/artifact/com.microsoft.azure/azure-keyvault
+[api_documentation]: not-valid-link
+[azkeyvault_docs]: https://docs.microsoft.com/en-us/azure/key-vault/
+[jdk]: https://docs.microsoft.com/en-us/java/azure/java-supported-jdk-runtime?view=azure-java-stable
+[maven]: https://maven.apache.org/
+[azure_subscription]: https://azure.microsoft.com/en-us/free/
+[azure_keyvault]: https://docs.microsoft.com/en-us/azure/key-vault/quick-create-portal
+[azure_cli]: https://docs.microsoft.com/cli/azure
+[rest_api]: https://docs.microsoft.com/en-us/rest/api/keyvault/
+[azkeyvault_rest]: https://docs.microsoft.com/en-us/rest/api/keyvault/
+[azure_create_application_in_portal]:https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-create-service-principal-portal
+[azure_keyvault_cli]:https://docs.microsoft.com/en-us/azure/key-vault/quick-create-cli
+[azure_keyvault_cli_full]:https://docs.microsoft.com/en-us/cli/azure/keyvault?view=azure-cli-latest
+[secrets_samples]:https://github.com/g2vinay/azure-sdk-for-java/tree/kv-secrets-samples-and-readme/keyvault/client/secrets/src/samples/java
+[sample_helloWorld]:https://github.com/g2vinay/azure-sdk-for-java/blob/kv-secrets-samples-and-readme/keyvault/client/secrets/src/samples/java/HelloWorld.java
+[sample_helloWorldAsync]:https://github.com/g2vinay/azure-sdk-for-java/blob/kv-secrets-samples-and-readme/keyvault/client/secrets/src/samples/java/HelloWorldAsync.java
+[sample_list]:https://github.com/g2vinay/azure-sdk-for-java/blob/kv-secrets-samples-and-readme/keyvault/client/secrets/src/samples/java/ListOperations.java
+[sample_listAsync]:https://github.com/g2vinay/azure-sdk-for-java/blob/kv-secrets-samples-and-readme/keyvault/client/secrets/src/samples/java/ListOperationsAsync.java
+[sample_BackupRestore]:https://github.com/g2vinay/azure-sdk-for-java/blob/kv-secrets-samples-and-readme/keyvault/client/secrets/src/samples/java/BackupAndRestoreOperations.java
+[sample_BackupRestoreAsync]:https://github.com/g2vinay/azure-sdk-for-java/blob/kv-secrets-samples-and-readme/keyvault/client/secrets/src/samples/java/BackupAndRestoreOperationsAsync.java
+[sample_ManageDeleted]:https://github.com/g2vinay/azure-sdk-for-java/blob/kv-secrets-samples-and-readme/keyvault/client/secrets/src/samples/java/ManagingDeletedSecrets.java
+[sample_ManageDeletedAsync]:https://github.com/g2vinay/azure-sdk-for-java/blob/kv-secrets-samples-and-readme/keyvault/client/secrets/src/samples/java/ManagingDeletedSecretsAsync.java
```

Comment:
Need to update all of these links to remove 'g2vinay' and link into the official repo instead.

Reply:
updated